README.md

```diff
@@ -5,12 +5,25 @@ JavaScript client library for Kill Bill.
 
 It is designed to run both on the client side as well as on the server side (e.g. using node.js).
 
-Tests
------
+Codegen
+-
+Code is generated using [openapi-codegen](https://github.com/OpenAPITools/openapi-generator) typescript-axios template. 
 
-Use node.js to run the tests:
+To regenerate, overwrite kbswagger.yaml and `npm run codegen && npm run build`
 
-```
-npm install
-npm test
-```
+Usage
+-
+``` javascript
+const killbill = require('killbill');
+const config = new killbill.Configuration({
+    username: "admin"
+    password: "password",
+    apiKey: "bob",
+    accessToken: "lazar",
```

Comment:
`accessToken` is a bit confusing, I'd suggest using `apiSecret` instead.

Reply:
Yeah, that is the default in the template for some reason... let me see if I can override it somehow.
https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/typescript-axios/configuration.mustache#L9

Reply:
Alright think I figured this out.. not pretty but something like this will work.. actually accessToken will not be used at all in our case.  I will see if there is a cleaner way to do this and update the test/documentation.

`apiKey: (k: string) => {if (k === 'X-Killbill-ApiKey') return 'bob'; if (k === 'X-Killbill-ApiSecret') return  'lazar'},`